rmgpy/data/base.py

```diff
@@ -322,22 +322,26 @@ def save_dictionary(self, path):
                 f.write(species_dict[label].molecule[0].to_adjacency_list(label=label, remove_h=False))
                 f.write('\n')
 
-    def save(self, path):
+    def save(self, path, reindex=True):
         """
         Save the current database to the file at location `path` on disk. 
         """
         try:
             os.makedirs(os.path.dirname(path))
         except OSError:
             pass
-        entries = self.get_entries_to_save()
+        if reindex is True:
+            entries = self.get_entries_to_save()
+        else:
+            entries = self.entries.values()
+
 
         f = codecs.open(path, 'w', 'utf-8')
         f.write('#!/usr/bin/env python\n')
         f.write('# encoding: utf-8\n\n')
         f.write('name = "{0}"\n'.format(self.name))
-        f.write('shortDesc = "{0}"\n'.format(self.short_desc))
-        f.write('longDesc = """\n')
+        f.write('shortDesc = u"{0}"\n'.format(self.short_desc))
```

Comment:
Not sure if we need this string to be unicode 🤔. Is this the reason why the tests are failing?